app/_data/docs_nav_ee_0.35-x.yml

```
@@ -0,0 +1,210 @@
+
+- title: Getting Started
+  url: /getting-started/
+  items:
+    - text: Overview
+      url: /getting-started/
+    - text: How to Start Kong Enterprise
+      url: /getting-started/start-kong
+    - text: How to Add a Workspace
+      url: /getting-started/add-workspace
+    - text: How to Add a Role and Permissions
+      url: /getting-started/add-role
+    - text: How to Add an Admin
+      url: /getting-started/add-admin
+    - text: How to Add a Service and Route
+      url: /getting-started/add-service
+    - text: How to Enable a Plugin
+      url: /getting-started/enable-plugin
+    - text: How to Add a Consumer
+      url: /getting-started/add-consumer
+    - text: How to Enable the Dev Portal
+      url: /getting-started/enable-dev-portal
+
+- title: Admin API
+  url: /admin-api/
+  items:
+  - text: Workspaces
+    url: /admin-api/workspaces/reference
+  - text: RBAC
+    url: /admin-api/rbac/reference
+  - text: Admins
+    url: /admin-api/admins/reference
+    items:
+      - text: API Reference
+        url: /admin-api/admins/reference
+      - text: Examples
+        url: /admin-api/admins/examples
+  - text: Vitals
+    url: /admin-api/vitals
+    items:
+      - text: Vitals Overview
+        url: /admin-api/vitals
+      - text: Kong Vitals with InfluxDB
+        url: /admin-api/vitals/vitals-influx-strategy
+      - text: Kong Vitals with Prometheus
+        url: /admin-api/vitals/vitals-prometheus-strategy
+  - text: Audit Logging
+    url: /admin-api/audit-log
+
+- title: Kong Manager
+  items:
+    - text: Overview
+      url: /kong-manager/overview
+    - text: Networking
+      url: /kong-manager/networking/configuration
+      items:
+        - text: Default and Custom Configuration
+          url: /kong-manager/networking/configuration
+        - text: Configuring Kong Manager to Send Email
+          url: /kong-manager/networking/email
+    - text: Securing Kong Manager
+      url: /kong-manager/security
+    - text: Authentication
+      url: /kong-manager/authentication/super-admin
+      items:
+        - text: How to Create a Super Admin
+          url: /kong-manager/authentication/super-admin
+        - text: Basic Auth
+          url: /kong-manager/authentication/basic
+        - text: LDAP
+          url: /kong-manager/authentication/ldap
+        - text: OIDC
+          url: /kong-manager/authentication/oidc
+        - text: Sessions
+          url: /kong-manager/authentication/sessions
+    - text: Administration
+      url: /kong-manager/administration/workspaces/workspaces
+      items:
+        - text: Workspaces
+          url: /kong-manager/administration/workspaces/workspaces
+        - items:
+          - text: Workspaces in Kong Manager
+            url: /kong-manager/administration/workspaces/workspaces
+          - text: How to Create a New Workspace
+            url: /kong-manager/administration/workspaces/create-workspace
+          - text: How to Update or Delete a Workspace
```

@levfishbluefish your call on this one, just want to get rid of wrapping.

```suggestion
          - text: How to Update a Workspace
```

---

will need non trivial style updates to prevent wrapping so will accept for now